src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  font-family: "Arial";
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin-top: 1em;
+  margin-bottom: 1em;
+}
+
+a {
+  color: green;
+  font-size: 15px;
+  font-weight: normal;
+}
+
+.articles {
+  display: none;
+}
```

Comment:
What is reason for this?

Reply:
I had another part of the text with articles 17 etc, that's why I added this class style, forgotten to delete